tasks.py

```diff
@@ -157,17 +157,17 @@ def _detect_space(repo, branch=None, yes=False):
 
 
 SPACE_URLS = {
-    'dev': [('18f.gov', 'fec-dev-api')],
-    'stage': [('18f.gov', 'fec-stage-api')],
-    'prod': [('18f.gov', 'fec-prod-api')],
+    'dev': [('app.cloud.gov', 'fec-dev-api')],
+    'stage': [('app.cloud.gov', 'fec-stage-api')],
+    'prod': [('app.cloud.gov', 'fec-prod-api')],
```

Comment:
These will have to change in the future as we get the domain configuration set in GovCloud, right @LindsayYoung?